lib/WebSocketTransport.js

```diff
@@ -29,6 +29,12 @@ class WebSocketTransport {
         perMessageDeflate: false,
         maxPayload: 256 * 1024 * 1024, // 256Mb
       });
+
+      /* WebSocket is not assignable to type WebSocket due to a
+       * missing dispatchEvent() method which the ws library
```

Comment:
Can we file a bug at either the `ws` library or its types to get that fixed?

Reply:
It's fixed in ws: https://github.com/websockets/ws/commit/cec1e1cf44f2aeadbe7aa271c46e00cee245a135

But upgrading `ws` in this project didn't help. I then thought about upgrading `@types/ws` and that didn't help either. So I _think_ the problem is in `@types/ws` but I'm not 100% clear.